lib/oli_web/live/workspaces/course_author/overview_live.ex

```diff
@@ -51,35 +44,32 @@ defmodule OliWeb.Workspaces.CourseAuthor.OverviewLive do
     # Subscribe to any project export progress updates for this project
     Subscriber.subscribe_to_project_export_status(project.slug)
 
-    socket =
-      assign(socket,
-        ctx: ctx,
-        breadcrumbs: [Breadcrumb.new(%{full_title: "Project Overview"})],
-        active: :overview,
-        collaborators: Accounts.project_authors(project),
-        activities_enabled: Activities.advanced_activities(project, is_admin?),
-        can_enable_experiments: is_admin? and Oli.Delivery.Experiments.experiments_enabled?(),
-        is_admin: is_admin?,
-        changeset: Project.changeset(project),
-        latest_published_publication: latest_published_publication,
-        publishers: Inventories.list_publishers(),
-        resource_title: project.title,
-        resource_slug: project.slug,
-        attributes: project.attributes,
-        language_codes: Oli.LanguageCodesIso639.codes(),
-        license_opts: cc_options,
-        collab_space_config: collab_space_config,
-        revision_slug: revision_slug,
-        latest_publication: latest_publication,
-        notes_config: %{},
-        project_export_status: project_export_status,
-        project_export_url: project_export_url,
-        project_export_timestamp: project_export_timestamp,
-        active_workspace: :course_author,
-        active_view: :overview
-      )
-
-    {:ok, socket}
+    {:ok,
+     assign(socket,
+       ctx: ctx,
+       active: :overview,
+       collaborators: Accounts.project_authors(project),
+       activities_enabled: Activities.advanced_activities(project, is_admin?),
+       can_enable_experiments: is_admin? and Experiments.experiments_enabled?(),
+       is_admin: is_admin?,
+       changeset: Project.changeset(project),
+       latest_published_publication: latest_published_publication,
+       publishers: Inventories.list_publishers(),
+       resource_title: project.title,
+       resource_slug: project.slug,
+       attributes: project.attributes,
+       language_codes: LanguageCodesIso639.codes(),
+       license_opts: cc_options,
+       collab_space_config: collab_space_config,
+       revision_slug: revision_slug,
+       latest_publication: latest_publication,
+       notes_config: %{},
+       project_export_status: project_export_status,
+       project_export_url: project_export_url,
+       project_export_timestamp: project_export_timestamp,
+       active_workspace: :course_author,
+       active_view: :overview
```

Comment:
NIT: `active` and `active_view` have the same value, so I wondered if we could remove one.

Reply:
Fixed!